chart/elastalert2/values.yaml

```diff
@@ -7,6 +7,9 @@ revisionHistoryLimit: 5
 # Default internal between alert checks against the elasticsearch datasource, in minutes
 runIntervalMins: 1
 
+# Enable/disabe subdirectory scanning for rules
+scanSubdirectories: false
```

Comment:
According to the docs, this option is true by default, which should mean that existing helm installations are already scanning subdirs: https://elastalert2.readthedocs.io/en/latest/elastalert.html

Changing this to default as False will break existing installations that upgrade. I think the defaults should match, which means this default value should be `true` here.